src/components/Sections/SectionList/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import './index.scss';
+
+const SectionList = ({
+  name,
+  description,
+  keywords,
+  url,
+  roles,
+  entity,
+  type,
+  startDate,
+  endDate,
+  highlights,
+  position,
+  summary,
+  organization,
+  sectionName,
+  reference,
+  language,
+  fluency,
+}) => {
+  const _highlights = highlights?.map((data, index) => {
+    return <li key={index} dangerouslySetInnerHTML={{ __html: data }}></li>;
+  });
+
+  const _keywords = keywords?.map((data, index) => {
+    return <li key={index} dangerouslySetInnerHTML={{ __html: data }}></li>;
+  });
+
+  const _roles = roles?.map((data, index) => {
+    return <li key={index} dangerouslySetInnerHTML={{ __html: data }}></li>;
+  });
+  if (
+    sectionName == 'Work Experience' ||
+    sectionName == 'Volunteer Work' ||
+    sectionName == 'Projects'
```

Comment:
If instead of directly comparing with title. I would suggest we should give this some template name which contains `summary along with hightlights` . So if in props it is passed, it means we have to render this layout. We need to think this as specific layout that should _not be aligned with title instead it works on template name_. If we pass that template it would always render like this.

Better way of writing this:
```suggestion
   const sectionHighlightsAllowed = ['Work Experience', 'Volunteer Work','Projects']
    if(sectionHighlightsAllowed.includes(sectionName))
```

If later in future we would want to add another field we simply add it to `sectionHighlightsAllowed` array.

Think template way also if you can do it in that way. Otherwise refactor with above one code